src/client/App.test.js

```diff
@@ -0,0 +1,11 @@
+import App from "./App";
+import React from "react";
+import { shallow } from "enzyme";
+
+describe("<App />", () => {
+  it("renders application", () => {
+    const wrapper = shallow(<App />);
+
+    expect(wrapper).toMatchSnapshot();
```

Comment:
I don't think this test will be useful for very long. I only included it here to make sure snapshot testing works and to provide an example.